From 46f475664d66e25f0cc4212786c441f929a91d05 Mon Sep 17 00:00:00 2001
From: Emmanuele Bassi <ebassi@gnome.org>
Date: Tue, 24 Oct 2017 16:58:06 +0100
Subject: Add a return value

When building with G_DISABLE_ASSERT, the g_assert_not_reached()
statement won't do anything, and we're going to fall through, and the
compiler will emit a warning that we're not returning anything from a
function with a return value.
---
 gdk/wayland/gdkwindow-wayland.c | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/gdk/wayland/gdkwindow-wayland.c b/gdk/wayland/gdkwindow-wayland.c
index bb15eb1..2d738a6 100644
--- a/gdk/wayland/gdkwindow-wayland.c
+++ b/gdk/wayland/gdkwindow-wayland.c
@@ -1617,6 +1617,9 @@ rect_anchor_to_anchor (GdkGravity rect_anchor)
     default:
       g_assert_not_reached ();
     }
+
+  return (ZXDG_POSITIONER_V6_ANCHOR_TOP |
+          ZXDG_POSITIONER_V6_ANCHOR_LEFT);
 }
 
 static enum zxdg_positioner_v6_gravity
@@ -1650,6 +1653,9 @@ window_anchor_to_gravity (GdkGravity rect_anchor)
     default:
       g_assert_not_reached ();
     }
+
+  return (ZXDG_POSITIONER_V6_GRAVITY_BOTTOM |
+          ZXDG_POSITIONER_V6_GRAVITY_RIGHT);
 }
 
 static GdkWindow *

